build-tools/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -55,7 +55,7 @@ public String toString() {
     private final Property<Boolean> failIfUnavailable;
     private final Property<Boolean> preferArchive;
     private final ConfigurableFileCollection extracted;
-    private Action<ElasticsearchDistribution> distributionFinalizer;
+    private transient Action<ElasticsearchDistribution> distributionFinalizer;
```

Comment:
This is only used when configuring the distribution. Once this is configured (and cached by the configuration cache) this is no longer required.